tests/model_inheritance/models.py

```diff
@@ -206,3 +206,14 @@ class SecondParent(CommonAncestor):
 
 class CommonChild(FirstParent, SecondParent):
     pass
+
+
+class ItalianRestaurantManyParents(ItalianRestaurant, Place):
+    place_ptr_two = models.OneToOneField(
+        Place, on_delete=models.CASCADE, parent_link=True
+    )
+
+
+class ItalianRestaurantManyParentsAbstract(ItalianRestaurant, Place):
+    class Meta:
+        abstract = True
```

Comment:
Why this is marked as `abstract`?

Reply:
Otherwise test will fail, as it will give conflict of `place_ptr` on check.

Reply:
You can move it directly to the test.